browser/components/GroupsTable.tsx

```diff
@@ -70,26 +70,70 @@ class GroupsTable extends React.Component<Props, State> {
     this.setState({
       alert: (
         <SweetAlert
-          type="input"
-          showCancel={true}
-          inputType="month"
-          title="Report Month"
+          title="Report Date"
           onConfirm={this.downloadReport}
-          onCancel={this.hideAlert}
-          cancelBtnBsStyle="warning"
+          showConfirm={false}
         >
           Please select the specific month and year for the report.
+          <form onSubmit={this.downloadReport}>
+            <div className="form-row">
+              <div className="form-group col">
+                <label htmlFor="month">Select month</label>
+                <select className="form-control" id="month">
+                  <option value="01">January</option>
+                  <option value="02">February</option>
+                  <option value="03">March</option>
+                  <option value="04">April</option>
+                  <option value="05">May</option>
+                  <option value="06">June</option>
+                  <option value="07">July</option>
+                  <option value="08">August</option>
+                  <option value="09">September</option>
+                  <option value="10">October</option>
+                  <option value="11">November</option>
+                  <option value="12">December</option>
+                </select>
+              </div>
+              <div className="form-group col">
+                <label htmlFor="year">Select year</label>
+                <select className="form-control" id="year">
+                  <option>2018</option>
```

Comment:
What if instead of hard coded years you request the oldest contribution from the DB and then generate a list of years to select from? That way we don't have to hard code anything and we ensure that we always have options for the oldest entries.

Reply:
Sounds good! Will do.